src/core/public/overlays/banners/banners_list.tsx

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import { Observable } from 'rxjs';
+
+import { OverlayBanner } from './banners_service';
+
+interface Props {
+  banners$: Observable<OverlayBanner[]>;
+}
+
+/**
+ * BannersList is a list of "banners". A banner something that is displayed at the top of Kibana that may or may not
+ * disappear.
+ *
+ * Whether or not a banner can be closed is completely up to the author of the banner. Some banners make sense to be
+ * static, such as banners meant to indicate the sensitivity (e.g., classification) of the information being
+ * represented.
+ */
+export const BannersList: React.FunctionComponent<Props> = ({ banners$ }) => {
+  const [banners, setBanners] = useState<OverlayBanner[]>([]);
+  useEffect(() => {
```

Comment:
can we switch to tested `useObservable` hook from https://github.com/restrry/kibana/blob/2cba693b8092916795c70512b0d959332f91abe3/src/plugins/kibana_react/public/util/use_observable.ts ?

Reply:
I don't think core should depend on any plugin code, even if it's just static code like this. Also I'm not sure why that hook uses `useLayoutEffect` instead of `useEffect`?